eng/Versions.props

```diff
@@ -107,6 +107,7 @@
     <!-- Testing -->
     <MicrosoftNETTestSdkVersion>16.7.0-preview-20200608-10</MicrosoftNETTestSdkVersion>
     <MicrosoftDotNetXHarnessTestRunnersXunitVersion>1.0.0-prerelease.20305.1</MicrosoftDotNetXHarnessTestRunnersXunitVersion>
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.20308.1</MicrosoftDotNetXHarnessCLIVersion>
```

Comment:
We should add this to `Versions.Details.xml` in order to get auto-updates for it.